src/Swagger/SchemaFormatter/JsonSchemaFormatter.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Swagger\SchemaFormatter;
+
+use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
+
+class JsonSchemaFormatter implements SchemaFormatterInterface
```

Comment:
Should be renamed `DefaultSchemaFormatter`, cause it can it's the default one, also used for XML and so on.